### ISSUE TITLE
UI: Improvement - make icons more accessible

### ISSUE THREAD

**Issue by damms005** [jest-community/vscode-jest#881]
### Environment

1. `vscode-jest version`: v4.6.0
2. `node -v`: v18.3.0
3. `npm -v`: 8.11.0
4. `npm ls jest` :
```
└─┬ @quasar/quasar-app-extension-testing-unit-jest@3.0.0-alpha.10
  ├── jest@26.6.3
  ├─┬ ts-jest@26.5.6
  │ └── jest@26.6.3 deduped
  └─┬ vue-jest@5.0.0-alpha.10
    └── jest@26.6.3 deduped
```
6. your vscode-jest settings if customized: N/A
7. Operating system: Ubuntu Linux 20.04

### Prerequisite
- are you able to run jest test from the command line? Yes
- how do you run your tests from the command line? `npm run test`

### Steps to Reproduce

Just a bit of UI improvement: 
When tests are listed in the sidebar, the icons are listed to the right of the test names. Consequently, when the test names are long enough, it pushes the icons way to the right, to the extent that one is required to always scroll to the right to access the icons, as shown in the screenshots below:

> Default view:

![jest-test-view](https://user-images.githubusercontent.com/9839355/173563997-919511c9-66fe-45b7-8fb6-b4df91f62d79.png)

> Scrolled view:

![jest-test-view-scrolled](https://user-images.githubusercontent.com/9839355/173564247-fed5abc7-6cbf-40b6-9f86-5db9382219ea.png)

> **Note**
> You can see as indicated in the arrows above that I had to scroll all the way to the right to access the icons for any test with long names

### Relevant Debug Info

N/A

### Expected Behavior

Icons should be visible without needing to scroll all the way to the right

### Actual Behavior

When test names are long enough, icons are **not** visible without scrolling all the way to the right

**Comment by damms005** [comment]
My good kind sir @connor4312, I hope this gets a little bit of your attention.

It seems, from my little finding, that all is needed here is to simply position the `ul.actions-container` items relative to the containing `div.monaco-list-row`

**Comment by connor4312** [comment]
Yes, I plan to do this (tracked in https://github.com/microsoft/vscode/issues/152622). PR's also welcome 🙂

**Comment by damms005** [comment]
Fixed by [#153838](https://github.com/microsoft/vscode/pull/153838)
